_scalingo_api/index.md

```diff
@@ -298,6 +298,15 @@ fmt.Println(date)
 
 --- row ---
 
+# Rate Limit
+
+In order to prevent abuse and denial-of-service attacks, the number of API requests you can
+make on a specific amount of time is limited. It ensures that the API remains available for all users.
+
+The limit is 60 requests per minutes. If you exceed that limit, the API will returns HTTP status 429: Too Many Requests.
```

Comment:
issue: as per our documentation about documentation writing (https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea), we usually don't use future tense 

```suggestion
The limit is 60 requests per minute. If you exceed that limit, the API returns HTTP status 429: Too Many Requests.
```